src/button.hpp

```diff
@@ -0,0 +1,28 @@
+#include <SFML/Graphics.hpp>
+
+class Button {
+public:
+	Button (std::string textString,
+		   	sf::RenderWindow & win,
+		   	sf::Color backgroundColor,
+		   	sf::Color textColor,
+		   	std::string fontAddress,
+		   	int characterSize );
+	void draw();
+	void setPosition (float x, float y, float width, float height);
```

Comment:
`setPosition()` takes button dimensions, too? Not good name, another thing to change - name or function.